x-pack/plugins/cases/public/components/create/flyout/create_case_flyout.tsx

```diff
@@ -87,6 +89,7 @@ export const CreateCaseFlyout = React.memo<CreateCaseFlyoutProps>(
             <EuiTitle size="m">
               <h2>{i18n.CREATE_CASE_TITLE}</h2>
             </EuiTitle>
+            {optionalContent && optionalContent}
```

Comment:
Could we add two tests:

- Checks that the content gets rendered when it is a react node
- Checks that the content does not get rendered when it is undefined